examples/ensemble/plot_voting_regressor.py

```diff
@@ -5,15 +5,21 @@
 
 .. currentmodule:: sklearn
 
-Plot individual and averaged regression predictions for Boston dataset.
-
-First, three exemplary regressors are initialized
-(:class:`~ensemble.GradientBoostingRegressor`,
+Regression is a method to predict the average value of the target y from the
+given data X. It is possible to use different type of regressors to predict the
```

Comment:
I think the first sentence is not very clear. Maybe something like 

> A voting regressor is an ensemble meta-estimator that fits base regressors each on the whole dataset. It, then, averages the individual predictions to form a final prediction.

(taken from the API doc of `VotingRegressor`).